### PR TITLE
Feat/special instruction

### PR DESCRIPTION
**Pull request by rizkybaihaqy**
## Type of changes
 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

special instruction from backend is not used

## What is the new behavior?

add special instruction instruction from backend inside dropzone for:
- all analytics
- face occlusion and attributes
- ocr receipt

add box to ekyc special instruction

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

screenshots:
image
image
same color as arrow down icon

image
image
font got bigger. can't miss that one

### CODE REVIEW

**Comment by sonadztux** [comment]
@rizkybaihaqy kalimat di drag & drop box Face Occlusion Attribute samain kaya OCR Receipt ky

**Comment by rizkybaihaqy** [comment]
> @rizkybaihaqy kalimat di drag & drop box Face Occlusion Attribute samain kaya OCR Receipt ky

Eh emang beda ya? Ga gw ganti per page kok. Gw ganti langsung dari komponen dropzonenya

File: frontend/app/components/modules/DropzoneOptions/DropzoneOptions.tsx [hunk 1]
```diff
@@ -107,7 +108,9 @@ export const DropzoneOptions = ({
           <input {...getInputProps()} />
           {photos.length === 0 ? (
             <aside className={styles.instruction}>
-              <p>Drag and drop your image here, or click to select image</p>
+              <p>{`Click or drag and drop image here or simply click on the given examples. Image is limited to ${getFileSizeWithUnit(
+                maxSize
```
Comment (gregoriusjimmy) [inline frontend/app/components/modules/DropzoneOptions/DropzoneOptions.tsx:112]: perlu di set props` maxImageSize,  acceptedFileFormat` untuk component yang pake DropzoneOptions ga skrng @williammfu ?

File: frontend/app/components/modules/DropzoneOptions/DropzoneOptions.tsx [hunk 2]
```diff
@@ -107,7 +108,9 @@ export const DropzoneOptions = ({
           <input {...getInputProps()} />
           {photos.length === 0 ? (
             <aside className={styles.instruction}>
-              <p>Drag and drop your image here, or click to select image</p>
+              <p>{`Click or drag and drop image here or simply click on the given examples. Image is limited to ${getFileSizeWithUnit(
```
Comment (gregoriusjimmy) [inline frontend/app/components/modules/DropzoneOptions/DropzoneOptions.tsx:111]: tolong skalian gantiin max sizenya @rizkybaihaqy di file [analytic_name].tsx jadi 3mb

Comment (rizkybaihaqy) [reply frontend/app/components/modules/DropzoneOptions/DropzoneOptions.tsx:111]: nanti gw buat branch baru aja

**Review by gregoriusjimmy — APPROVED** [review]

**Review by sonadztux — APPROVED** [review]

**Review by williammfu — APPROVED** [review]
lgtm

**Pull request merged**